Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/Validation/EconomizerMultiZone_Mod_DamLim.mo

```diff
@@ -0,0 +1,186 @@
+    annotation (Line(points={{-19,90},{-10,90},{-10,-22},{18,-22},{18,-30},{99,-30}}, color={0,0,127}));
+  connect(VOut_flowMinSet.y, economizer1.VOut_flowMinSet)
+    annotation (Line(points={{-19,50},{-12,50},{-12,-24},{16,-24},{16,-32},{99,-32}}, color={0,0,127}));
+  connect(FanStatus.y, economizer1.uSupFan)
+    annotation (Line(points={{-59,-80},{20,-80},{20,-34},{99,-34}}, color={255,0,255}));
+  connect(OperationMode.y, economizer1.uOperationMode)
+    annotation (Line(points={{-99,-100},{22,-100},{22,-36},{99,-36}}, color={255,127,0}));
+  connect(FreProSta.y, economizer1.uFreProSta)
+    annotation (Line(points={{-59,-120},{26,-120},{26,-40},{99,-40}}, color={255,127,0}));
+  connect(ZoneState.y, economizer1.uZoneState)
+    annotation (Line(points={{-99,-60},{24,-60},{24,-38},{99,-38}}, color={255,127,0}));
+  connect(hOutBelowCutoff.y, economizer1.hOut)
+    annotation (Line(points={{-99,20},{0,20},{0,-22},{99,-22}}, color={0,0,127}));
+  connect(hOutCut.y, economizer1.hOutCut)
+    annotation (Line(points={{-99,-20},{0,-20},{0,-24},{99,-24}}, color={0,0,127}));
+  annotation (
+    experiment(StopTime=900.0, Tolerance=1e-06),
+  __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/Experimental/OpenBuildingControl/ASHRAE/G36/CompositeSequences/Validation/EconomizerMultiZone_Mod_DamLim.mos"
+    "Simulate and plot"),
+  Icon(graphics={
+        Ellipse(lineColor = {75,138,73},
+                fillColor={255,255,255},
+                fillPattern = FillPattern.Solid,
+                extent={{-100,-100},{100,100}}),
+        Polygon(lineColor = {0,0,255},
+                fillColor = {75,138,73},
+                pattern = LinePattern.None,
+                fillPattern = FillPattern.Solid,
+                points={{-36,58},{64,-2},{-36,-62},{-36,58}})}), Diagram(
+        coordinateSystem(preserveAspectRatio=false, extent={{-140,-160},{140,160}}),
+        graphics={
+        Rectangle(
+          extent={{-136,-44},{-44,-156}},
+          lineColor={0,0,0},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid),
+                  Text(
+          extent={{52,106},{86,84}},
+          lineColor={28,108,200}),
+        Text(
+          extent={{-128,-130},{-44,-158}},
+          lineColor={0,0,0},
+          horizontalAlignment=TextAlignment.Left,
+          fontSize=9,
+          textString="Enable both damper limit 
+and modulation control loops"),
+        Text(
+          extent={{100,4},{136,-16}},
+          lineColor={0,0,0},
+          horizontalAlignment=TextAlignment.Left,
+          fontSize=8,
+          textString="Validate damper modulation
+(example without 
+enthalpy measurement)"),
+        Text(
+          extent={{20,46},{56,26}},
+          lineColor={0,0,0},
+          horizontalAlignment=TextAlignment.Left,
+          fontSize=8,
+          textString="Economizer fully enabled - 
+validate damper position limits")}),
+    Documentation(info="<html>
```

Comment:
In the future, please don't use spaces in the info section at the start of each line.